vault.js

```diff
@@ -1,4 +1,31 @@
 'use strict';
 module.exports = function() {
   
+  let vault = {};
+  
+  const setValue = function(key,value) {
+    if (!key || !value) {
+      return null;
+    } else {
+      vault.key = value;
+    }
+  }
+
+  const getValue = function(key) {
+    // // alternatively
```

Comment:
Remove commented code. If you ever need to look at this, you can always look in the commit history to see how you're done it in the past.